README.md

```diff
@@ -1,2 +1,16 @@
 # similar-npm-packages
+
 Similar packages suggestions for the NPM ecosystem
+
+## Install
+
+```
+npm install similar-npm-packages --save
```

Comment:
Do you want me to reserve this name?

Reply:
I do)

Reply:
BTW: Are we going to have NodeJS only packages here? Maybe `similar-npm-packages` isn't the best name if we're going to work with browsers packages only.

Reply:
We have some node.js packages in recommendations as well. Folks in the electron community find it useful to see sizes for these. Let's be stack agnostic for now.

Reply:
Alight. We should "book" the name then)

Reply:
`similar-npm-packages` is ours 🥳

Reply:
Nice!